integration-tests/src/tests.rs

```diff
@@ -184,6 +184,19 @@ macro_rules! common_tests { (($($attr:tt)*) $namespace:ident, $target:expr) => {
         assert_tests_fail_to_build( $target, "req-future-cargo-web-dev-dep" );
     }
 
+    $($attr)*
+    #[test]
+    fn mount_path() {
+        let cwd = crate_path( "mount-path" );
+        assert_builds( $target, "mount-path" );
+        // only applies to wasm32-unknown-unknown backend
+        if $target == Wasm32UnknownUnknown {
+            let build_dir = "release";
+            let output = cwd.join( "target" ).join( $target.to_str() ).join( build_dir ).join( "mount-path.js" );
+            assert_file_contains( output, "/custom/static/" );
+        }
+    }
```

Comment:
Please move this test to `wasm32-unknown-unknown`-specific tests since the functionality works only for that target anyway.